packages/hardhat-core/src/utils/source-names.ts

```diff
@@ -193,25 +186,19 @@ export function replaceBackslashes(str: string): string {
  * `p` doesn't exist. `p` MUST be in source name format.
  */
 async function getPathTrueCase(fromDir: string, p: string): Promise<string> {
-  const { trueCasePath } = await import("true-case-path");
-
   try {
-    const tcp = await trueCasePath(p, fromDir);
+    const absolute = path.join(fromDir, p);
+    const tcp = await getRealCase(absolute);
```

Comment:
This variable name is not very descriptive now that we are not using `trueCasePath` anymore.

Reply:
right. renamed it.